### PR TITLE
[liquidity redesign] Add space background image to liquidity page

### PR DESCRIPTION
**Pull request by Diogomartf**
### WIP 
_Me and @berteotti are working on updating the design of liquidity page and also the fonts and colors._

This is **work in progress** it's not finished, but we want to divide the work in smaller chunks and do things incrementally. 

We will do this smaller PRs and merge it on a "main" branch -> `feature/fonts-and-colors-liquidity-page`.

--- 

## Description

Add space background image to liquidity

image

## Test

1. go to liquitidy page
- check there is the space background image

### CODE REVIEW

File: src/components/SpaceBg/index.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+  z-index: 3;
+  min-height: calc(100vh - 340px);
+`
+
+const zoomOut = keyframes`
+    0% {transform: scale(1.2); opacity: 0;}
+    100% {transform: scale(1); opacity: 0.7;}
+`
+
+const StyledHero = styled.div`
```
Comment (berteotti) [inline src/components/SpaceBg/index.js:33]: Is there a way to not repeat this code? If think the same style is being used in the swap page / landing

Comment (berteotti) [reply src/components/SpaceBg/index.js:33]: This should apply to other repeating code in this file

**Comment by MilanVojnovic95** [comment]
Tested and looks ok to me.

File: src/components/SpaceBg/index.js (hunk 1 above)
Comment (Diogomartf) [reply src/components/SpaceBg/index.js:33]: We should the same Styled hero and background image but I think we should do it after nailing the right abstractions. They are currently different as the hero on swap page has a lot of other things.

**Review by berteotti — APPROVED** [review]

**Pull request merged**